src/models/applicationState.ts

```diff
@@ -44,6 +44,7 @@ export enum ErrorCode {
     ProjectUploadError = "projectUploadError",
     ProjectDeleteError = "projectDeleteError",
     SecurityTokenNotFound = "securityTokenNotFound",
+    ExportFormatNotFound = "exportFormatNotFound",
```

Comment:
When specifying a new error code you will also need to setup the localized strings for the error code.

Reply:
fixed